pubspec.yaml

```diff
@@ -29,7 +29,10 @@ dependencies:
   chewie_audio: ^1.2.0
 
   # Plugins for rendering the <svg> tag.
-  flutter_svg: ^0.21.0-nullsafety.0
+  flutter_svg: ^0.20.0-nullsafety.3
```

Comment:
Why do we need the downgrade here?

Reply:
flutter_math was constrained to 0.20.0, I changed the dependency to flutter_math_fork because flutter_math hasn't been maintained for a bit. It's essentially the same but has a hotfix for an error occurring on Flutter 2.0.0+.